ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__automated-release.yaml

```diff
@@ -122,6 +122,18 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-automated-release
     workflow: cucushift-installer-rehearse-gcp-ipi
+- as: aws-ipi-disruptive-f999
+  cron: 8 8 29 2 *
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: aws-autorelease-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      CATALOGSOURCE_NAME: auto-release-app-registry
+      TEST_IMPORTANCE: LEVEL0.*Disruptive.*
```

Comment:
We should be good to move the env `CATALOGSOURCE_NAME` and `TEST_IMPORTANCE` into the chain `openshift-e2e-test-qe-automated-release-disruptive`. In this way, we only need to define them once, rather than many times in every job, which is easier to maintain.

Reply:
@liangxia yes, your're right. default setting should be centralized. this change needs to be applied to automated-release chain as well. I'll do it in separate PR

Reply:
Ack

Reply:
I tried to override `CATALOGSOURCE_NAME` in chain `openshift-e2e-test-qe-automated-release-disruptive` but it is failed with below error.
```
parameter \"CATALOGSOURCE_NAME\" is overridden in [chain/openshift-e2e-test-qe-automated-release-disruptive] but not declared in any step
```
because this env var is declared in step `enable-qe-catalogsource` and it is referenced in installation workflow e.g.  `cucushift-installer-rehearse-aws-ipi-provision`, so I suppose it only can be overridden at job level

Reply:
@liangxia I applied the change that override the env var `TEST_IMPORTANCE` in chain `openshift-e2e-test-qe-automated-release-disruptive`

Reply:
@liangxia can you review again, thx